charts/portworx/v2.8.1/templates/_helpers.tpl

```diff
@@ -95,6 +95,35 @@ release: {{ .Release.Name | quote }}
 {{- end -}}
 {{- end -}}
 
+
+{{- define "px.getCSIProvisionerImage" -}}
+{{- if semverCompare "<=1.16.0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
sorry, I meant 1.16.x and earler - so the actual compare should be 
`<1.17.0`. The above will not work for any 1.16 patch - i.e. 1.16.7